.github/scripts/collect_layer_metrics.py

```
@@ -0,0 +1,211 @@
+
+import json
+import subprocess
+import sys
+from datetime import datetime, timezone
+from typing import Any, Dict, List, Optional
+
+
+def run_command(cmd: List[str]) -> str:
+    """Run a command and return its output"""
+    try:
+        result = subprocess.run(
```

Ruff (0.14.5)

`subprocess` call: check for execution of untrusted input

(S603)